packages/db-orchestrator/index.cjs

```diff
@@ -177,8 +176,8 @@ const runQueries = async function (routeHash, dev) {
 							populateColumnTypeMetadata(data, queryIndex, columnTypes);
 
 							readline.cursorTo(process.stdout, 0);
```

Comment:
Are the `readline.cursorTo(process.stdout, 0);`s still necessary?

Reply:
Ah likely not